### PR TITLE
renames metal sheets to iron sheets

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request

what the title says
removes unused CRAFTLATHE define that i noticed

## Why It's Good For The Game

its a specific type of metal, it shouldnt just be called generic "metal"
the reagent, ore and material datum are already called iron

## Changelog
:cl:
spellcheck: renames metal sheets to iron sheets
/:cl:

### CODE REVIEW

**Comment by ATH1909** [comment]
he did it

the absolute madman

**Comment by cacogen** [comment]
Thank God

File: strings/ion_laws.json [hunk 1]
```diff
@@ -652,8 +652,9 @@
         "HAIRDOS",
         "HELMETS",
         "HORNS",
-        "ID CARDS",
-        "INSULATED GLOVES",
+		"ID CARDS",
+		"INSULATED GLOVES",
```
Comment (ShizCalev) [inline strings/ion_laws.json:656]: fix spacing

File: strings/ion_laws.json [hunk 2]
```diff
@@ -652,8 +652,9 @@
         "HAIRDOS",
         "HELMETS",
         "HORNS",
-        "ID CARDS",
-        "INSULATED GLOVES",
+		"ID CARDS",
```
Comment (ShizCalev) [inline strings/ion_laws.json:655]: spacing

File: code/game/gamemodes/sandbox/h_sandbox.dm [hunk 3]
```diff
@@ -190,7 +190,7 @@ GLOBAL_VAR_INIT(hsboxspawn, TRUE)
 				new/obj/item/stack/sheet/rglass{amount=50}(usr.loc)
 
 			if("hsbmetal")
-				new/obj/item/stack/sheet/metal{amount=50}(usr.loc)
+				new/obj/item/stack/sheet/iron{amount=50}(usr.loc)
```
Comment (coiax) [inline code/game/gamemodes/sandbox/h_sandbox.dm:193]: There's an existing `/fifty` typepath isn't there?

File: code/game/machinery/doors/airlock.dm [hunk 4]
```diff
@@ -39,7 +39,7 @@
 #define AIRLOCK_EMAG	6
 
 #define AIRLOCK_SECURITY_NONE			0 //Normal airlock				//Wires are not secured
-#define AIRLOCK_SECURITY_METAL			1 //Medium security airlock		//There is a simple metal over wires (use welder)
+#define AIRLOCK_SECURITY_IRON			1 //Medium security airlock		//There is a simple iron over wires (use welder)
```
Comment (coiax) [inline code/game/machinery/doors/airlock.dm:42]:
```suggestion
#define AIRLOCK_SECURITY_IRON			1 //Medium security airlock		//There is a simple iron plate over wires (use welder)
```

**Comment by notfrying1pans** [comment]
iron out deez nuts gottem

File: code/game/objects/items/stacks/rods.dm [hunk 5]
```diff
@@ -58,9 +58,9 @@ GLOBAL_LIST_INIT(rod_recipes, list ( \
 			return
 
 		if(W.use_tool(src, user, 0, volume=40))
-			var/obj/item/stack/sheet/metal/new_item = new(usr.loc)
-			user.visible_message("<span class='notice'>[user.name] shaped [src] into metal with [W].</span>", \
-				"<span class='notice'>You shape [src] into metal with [W].</span>", \
+			var/obj/item/stack/sheet/iron/new_item = new(usr.loc)
+			user.visible_message("<span class='notice'>[user.name] shaped [src] into iron with [W].</span>", \
+				"<span class='notice'>You shape [src] into iron with [W].</span>", \
```
Comment (coiax) [inline code/game/objects/items/stacks/rods.dm:63]: Language used seems weird

> "You shape the iron rods into iron with the welder."

I don't have a better suggestion though 🤷

File: code/game/objects/structures/watercloset.dm [hunk 6]
```diff
@@ -9,7 +9,7 @@
 	var/cistern = 0			//if the cistern bit is open
 	var/w_items = 0			//the combined w_class of all the items in the cistern
 	var/mob/living/swirlie = null	//the mob being given a swirlie
-	var/buildstacktype = /obj/item/stack/sheet/metal //they're metal now, shut up
+	var/buildstacktype = /obj/item/stack/sheet/iron //they're metal now, shut up
```
Comment (coiax) [inline code/game/objects/structures/watercloset.dm:12]: I mean, they're iron now, surely? (Do we have to refactor meta comments when refactoring? Jesus.)

File: code/modules/cargo/exports.dm [hunk 7]
```diff
@@ -11,7 +11,7 @@ Crate cost is 500cr for a regular plasteel crate and 100cr for a large wooden on
 This is to avoid easy cargo points dupes.
 
 Credit dupes that require a lot of manual work shouldn't be removed, unless they yield too much profit for too little work.
-For example, if some player buys metal and glass sheets and uses them to make and sell reinforced glass:
+For example, if some player buys iron and glass sheets and uses them to make and sell reinforced glass:
 
 100 glass + 50 metal -> 100 reinforced glass
```
Comment (coiax) [inline code/modules/cargo/exports.dm:16]:
```suggestion
100 glass + 50 iron-> 100 reinforced glass
```

File: code/game/gamemodes/sandbox/h_sandbox.dm (hunk 3 above)
Comment (Fikou) [reply code/game/gamemodes/sandbox/h_sandbox.dm:193]:
```suggestion
				new/obj/item/stack/sheet/iron/fifty(usr.loc)
```

File: code/game/objects/structures/watercloset.dm (hunk 6 above)
Comment (Fikou) [reply code/game/objects/structures/watercloset.dm:12]:
```suggestion
	var/buildstacktype = /obj/item/stack/sheet/iron //they're iron now, shut up
```

File: code/game/objects/items/stacks/rods.dm (hunk 5 above)
Comment (cacogen) [reply code/game/objects/items/stacks/rods.dm:63]: iron sheets

Comment (Fikou) [inline code/game/objects/items/stacks/rods.dm:63]:
```suggestion
			user.visible_message("<span class='notice'>[user.name] shaped [src] into iron sheets with [W].</span>", \
				"<span class='notice'>You shape [src] into iron sheets with [W].</span>", \
```

File: code/game/objects/structures/watercloset.dm (hunk 6 above)
Comment (cacogen) [reply code/game/objects/structures/watercloset.dm]: not even clear what it means anyway

**Comment by LemonInTheDark** [comment]
affected please don't kill me

**Pull request merged**